best-practices/development-tools/azure/devops/deployment/azure-app-functions/azure-app-functions-deployment.md

```diff
@@ -0,0 +1,26 @@
+# Deployment using Azure DevOps
+
+## Pre-Deployment
+### Install AzureDevOps Extensions
+There are two extensions should be installed before in AzureDevOps [ReplaceTokens](https://marketplace.visualstudio.com/items?itemName=citrus-andriessen.cag-replace-tokens&targetId=af056fb1-e0c2-4678-9ac3-dee687452af2&utm_source=vstsproduct&utm_medium=ExtHubManageList).
```

Comment:
Same as above, I believe these two extensions should be required.

Reply:
Yah I noticed that and fixed it as well, Thanks again.